kubetest2-aks/README.md

```diff
@@ -0,0 +1,23 @@
+This is the kubetest2-aks repo to provision and delete aks clusters.
+
+## Commands
+Make kubetest2-aks binary
+```
+make install-deployer
+```
+
```

Comment:
Could you also add a list of environment variables that should be set before the following command?

Reply:
Updated.